demo/docbot/cluster_bootstrap.py

```diff
@@ -244,48 +248,49 @@ def initialize_ingestion_pipeline(client: MLClient):
 
 
 
-def initialize_index(client: MLClient) -> None:
+def initialize_index(client: MLClient, index="cohere-index", index_data=None) -> None:
```

Comment:
Here we can remove index_data and maybe we should lift the default pipeline as variable.